src/Silences/Api.elm

```diff
@@ -1,36 +1,37 @@
 module Silences.Api exposing (..)
 
 import Http
-import Silences.Types exposing (Silence, SilencesMsg(..), Msg(..))
-import Utils.Types exposing (Filter, ApiResponse(..))
+import Silences.Types exposing (Silence)
+import Utils.Types exposing (Filter, ApiResponse(..), ApiData)
 import Silences.Decoders exposing (..)
 import Silences.Encoders
 import Utils.Api exposing (baseUrl)
 import Utils.Filter exposing (generateQueryString)
+import Types exposing (Msg)
 
 
-getSilences : Filter -> Cmd Msg
-getSilences filter =
+getSilences : Filter -> (ApiData (List Silence) -> Msg) -> Cmd Msg
```

Comment:
Api requests do not need to know about the `Msg` type, you can make it lowercase and remove the dependency on the top level `import Types exposing (Msg)`.